src/org/testKitGen/TestDivider.java

```diff
@@ -226,21 +227,58 @@ private Map<String, Integer> getDataFromTRSS()  {
 		String level = getLevel();
 		Map<String, Integer> map = new HashMap<String, Integer>();
 		String URL = constructURL(impl, plat, group, level);
-		String command = "curl --silent --max-time 120 -L " + URL;
+		String osName = System.getProperty("os.name").toLowerCase();
+		String responseFilePath = new File("response.json").getAbsolutePath();
+		String command;
+
+		if (osName.contains("win")) {
+			command = "cmd.exe /c curl --silent --max-time 120 -L -k \"" + URL + "\" -o " + responseFilePath;
+		} else {
+			command = "curl --silent --max-time 120 -L -k "+ URL + " -o " + responseFilePath;
+		}
```

Comment:
A file is suggested to debug the problem. We should try to avoid using it if possible.

Reply:
I have removed the use of the file `response.json` in the code as suggested. 
Grinder links:
TARGET: sanity.functional
mac:
[42294](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42294/)
win:
[42295](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42295/)
aarch64_linux:
[#42296](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42296/)
s390x_linux:
[#42292](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42292/)

Reply:
Thanks @annaibm . It looks like the fix is to add quotes around `URL`. Do we need the following changes (L244 - L319)?

Reply:
I have removed changes from (L244 - L319) . The fix for constructing the curl command based on the operating system . Added quotes around the URL for Windows, while no quotes are used for Linux. Also added the -k option in the curl command.

Reply:
Grinder links:
x86_64_linux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42341/console

s390x_linux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42340/

ppc64_aix
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42338/console

aarch64_linux
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42344/console

x86_64_windows
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42343/consoleFull

x86_64_mac
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/42342/console